src/ir/datalog/BUILD

```diff
@@ -58,6 +58,21 @@ cc_library(
     ],
 )
 
+cc_library(
+    name = "raksha",
```

Comment:
Use a more specific name?

Reply:
Done, now is `raksha_relation_interface`, indicating that this library contains the data structures representing `.input`/`.output` relations that describe the interface to the Raksha analysis.